docs/features/theme.stories.mdx

```diff
@@ -16,12 +15,8 @@ import { Headline, SubHeadline, Body } from '../../packages/circuit-ui';
 
 # Theme
 
-<Intro>
-
 The theme used throughout Circuit UI comes from SumUp's design token library, [`@sumup/design-tokens`](Packages/design-tokens), which is a required peer dependency of `@sumup/circuit-ui`. Refer to its [documentation](Packages/design-tokens) for usage instructions.
 
-</Intro>
-
```

Comment:
Why did you remove the `Intro` here? I preferred the previous styling 😕

Reply:
Because nested `<p>` tags is invalid markup (and by extension a WCAG violation). I'm happy to look into a better styling solution for this `Intro` component like I mentioned [here](https://github.com/sumup-oss/circuit-ui/pull/1392#discussion_r794882086), but I think that we should start with strong foundations and valid HTML 🙂 

<img width="1443" alt="Screenshot 2022-01-29 at 16 07 32" src="https://user-images.githubusercontent.com/35560568/151666103-b75ebca3-86fd-4da5-8f18-4e7a5c67400f.png">

Reply:
Actually it's pretty straightforward—the `Intro` component should use the new `BodyLarge` 🙂

Reply:
Hm but actually, the problem is that in MDX if you do this:

```mdx
<Intro>

Intro text here

</Intro>
```

The text will be wrapped in a paragraph:

```html
<p> <!-- 👈 this is the `Intro` component -->
  <p>Intro text here</p>
</p>
```

---

And in contrast if you omit the line breaks:

```mdx
<Intro>Intro text here</Intro>
```

...then you can't include markdown links in the intro text (`[link](url)`), because it will be considered HTML. So in our case we could still link to the ^Packages/Design Tokens` page but we'd have to make this a "real" link which would cause a full page load

Reply:
How about rendering the Intro component as a `div`?

```html
<div> <!-- 👈 this is the `Intro` component -->
  <p>Intro text here</p>
</div>
```

Reply:
Yes this would work but I think that the child p styles would override the parent div styles (so the text would be typography.body.one). I can find a solution on Monday though (there's always !important 🙈)